packages/react-ui/src/widgets/legend/LegendCategories.js

```diff
@@ -121,8 +135,25 @@ function Row({ label, isMax, isStrokeColor, color = '#000' }) {
           <Box
             mr={1.5}
             component='span'
-            className={classes.circle}
-            style={isStrokeColor ? { borderColor: color } : { backgroundColor: color }}
+            className={[
+              classes.marker,
+              icon ? classes.markerIcon : classes.markerCircle
+            ].join(' ')}
+            style={
+              icon
+                ? {
+                    backgroundColor: color,
+                    maskRepeat: 'no-repeat',
+                    maskSize: 'cover',
+                    maskImage: `url(${icon})`,
+                    WebkitMaskRepeat: 'no-repeat',
```

Comment:
Is this really required? MaterialUI is supposed to detect & add vendor prefixes (https://v4.mui.com/styles/advanced/#css-prefixes)

Reply:
Didn't know about mui. This is not MUI, btw, this is generic react `style` property ...
For sure prefixes are needed with vanilla CSS: https://jsfiddle.net/ongwmLrd/2/
Will check it...

Reply:
It's some other magic mechanism, which understands Firefox 
![image](https://user-images.githubusercontent.com/1507542/178799596-ffe3a91e-8e2b-48a4-8fcd-8369eaf7a72d.png)
vs Chrome:
<img width="317" alt="image" src="https://user-images.githubusercontent.com/1507542/178799689-6f87c634-cad0-4df3-aeb6-e1c95d96163a.png">

and ... if we don't add `Webkit*`, it only looks like this:

![image](https://user-images.githubusercontent.com/1507542/178799979-61ef9b29-a371-45f0-9d95-6de616f07532.png)

In other words, for Chrome, supposedly unsupported properties aren't even emitted :/ Too many frameworks for me